### DISCUSSION TITLE
how to use npm run build in next js

### DISCUSSION THREAD

**Discussion by sromexs** [vercel/next.js#14083] [Help] [answered]
Hello guys i want to know in applications like react we use html output file to run frontend
but here after we build out application with
`npm run build`
what exactly we should do ?

**Comment by rafaelalmeidatk** [comment, marked as answer]
`next build` will build your files for the next  command which can be `next start` or `next export`. `next start` will start a Node.js server that will listen to requests and send your pages, this is the standard way to use Next.js since it has all the features. `next export` is for [Static HTML Export](https://nextjs.org/docs/advanced-features/static-html-export), it will output HTML files similar to what `create-react-app` does

**Reply by sromexs** [reply]
> `next build` will build your files for the next command which can be `next start` or `next export`. `next start` will start a Node.js server that will listen to requests and send your pages, this is the standard way to use Next.js since it has all the features. `next export` is for [Static HTML Export](https://nextjs.org/docs/advanced-features/static-html-export), it will output HTML files similar to what `create-react-app` does

Thanks Bro
I set `"export": "next export"` and after `"npm run build"` i send `npm run export` command and thats worked
Thank you
i send you an email please check that.
